.travis.yml

```diff
@@ -1,6 +1,13 @@
 language: python
 python:
-  - "2.7"
+  - 2.7
+sudo: true
+env:
+  - TOX_ENV=py27
+  - TOX_ENV=py34
+  - TOX_ENV=lint
+  - TOX_ENV=docs
 install:
-  - "pip install ."
-script: python setup.py test
+  - pip install tox
+script:
+  - tox -e $TOX_ENV
```

Comment:
Should we standardize on reporting these failure somewhere as well? I did slack reporting in autoapi: https://github.com/rtfd/sphinx-autoapi/pull/18/files#diff-354f30a63fb0907d4ad57269548329e3R13